pages/api/Schema/reports/expectedMonthlyTotal/expectedMonthlyTotal.graphql

```diff
@@ -0,0 +1,27 @@
+extend type Query {
+  expectedMonthlyTotalReport(accountListId: ID!): ExpectedMonthlyTotalReport!
+}
+
+type ExpectedMonthlyTotalReport {
+  received: [ExpectedMonthlyTotalDonation!]!
+  likely: [ExpectedMonthlyTotalDonation!]!
+  unlikely: [ExpectedMonthlyTotalDonation!]!
+  currency: String
+  currencySymbol: String
+}
```

Comment:
We're going to need a total of the expected monthly donations, and then subtotals for each of the three categories.  Do you think that calculation should be done in `dataHandler` and provided here, or should that be done in the component?

Reply:
I think that can be done here. Good point :)